Elasticsearch/Client/Info.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CommonBundle\Elasticsearch\Client;
+
+use EMS\CommonBundle\Contracts\Elasticsearch\Cluster\InfoInterface;
+
+final class Info implements InfoInterface
+{
+    /** @var string */
+    private $clusterName;
+    /** @var string */
+    private $clusterUuid;
+    /** @var string */
+    private $tagLine;
+    /** @var string */
+    private $nodeName;
+    /** @var string */
+    private $versionNumber;
+    /** @var string */
+    private $versionNumberLucene;
+    /** @var \DateTimeImmutable */
+    private $versionBuildDate;
+
+    public function __construct(array $info)
+    {
+        $this->clusterName = $info['cluster_name'];
+        $this->clusterUuid = $info['cluster_uuid'];
+        $this->tagLine = $info['tagline'];
+        $this->nodeName = $info['name'];
+        $this->versionNumber = $info['version']['number'];
+        $this->versionNumberLucene = $info['version']['lucene_version'];
+        $this->versionBuildDate = \DateTimeImmutable::createFromFormat('Y-m-d\TH:i:s.u\Z', $info['version']['build_date']);
```

Comment:
createFromFormat can return false or null on failure. Time time to create the wrapper around the custom error I created the other day?

Reply:
I know but I don't like it, but for now I will use it. Everything to make phpstan happy